src/storage/mvcc/reader/point_getter.rs

```diff
@@ -692,19 +692,19 @@ mod tests {
         let mut getter = new_point_getter(&engine, TimeStamp::max());
         let perf_statistics = PerfStatisticsInstant::new();
         must_get_value(&mut getter, b"foo", b"bar");
-        assert_eq!(perf_statistics.delta().0.internal_delete_skipped_count, 0);
+        assert_eq!(perf_statistics.delta().internal_delete_skipped_count, 0);
```

Comment:
Looks like it is now possible to record critical per-request latency metrics for troubleshooting fs related jitters, like Mutex time and reading SST file time without much cost. Do you plan to add them later?

Reply:
Yes. I'm going to record them.

Reply:
Yes. I'm going to record them.